woocommerce/google-listings-and-ads
# [Don't merge] Make step header configurable in SetupFreeListings.

_This is another take on https://github.com/woocommerce/google-listings-and-ads/pull/226 UI only part of 3. of https://github.com/woocommerce/google-listings-and-ads/pull/226#issuecomment-790215574_
:warning: This PR is dependent on https://github.com/woocommerce/google-listings-and-ads/pull/290
### Changes proposed in this Pull Request:

- Make step header configurable in `<Hero>` component. 
- Fix "STEP THREE"-> "STEP TWO" in Edit Campaign
- Make the `<TopBar>` a component shared by Onboarding and EditCampaign

Implements part of https://github.com/woocommerce/google-listings-and-ads/issues/156.



### Screenshots:

![image](https://user-images.githubusercontent.com/17435/110003272-7f188d00-7d16-11eb-9a28-f826aba32056.png)



### Detailed test instructions:

0. Check that the project builds (dependencies are resolved correctly for Onboarding)
1. Go to [the GLA Dashboard](https://gla1.test/wp-admin/admin.php?page=wc-admin&path=%2Fgoogle%2Fdashboard)
2. Click the "edit" button on a free listing
3. Check if the topbar and step header matches [Figma designs](https://www.figma.com/file/jZUpa8eTrnrK1Lwt2ry7zk/Native-Google-Integration?node-id=4162%3A0)



### Doubts:

1. TopBar in onboarding has a help button which is not present in Figma in this view. I guess, we assume that a merchant is fluent enough at this point and does not need help link. (cc @j111q ?)
2. TopBar in onboarding records track events on the back and help actions. I guess, it would generate too much noise and we don't need it here. (cc @jconroy ?)
